### PR TITLE
Incorporate Libusha Kelly Specific Aims comments and tweak \eject

### PR DESCRIPTION
**Pull request by jchodera**

### CODE REVIEW

**Comment by jchodera** [comment]
I incorporated some comments from Libusha Kelly on the Specific Aims, but it really needs a careful edits pass to make sure it matches our message from the Research Strategy.

Ready for review and merge.

File: latex/Specific Aims.tex [hunk 1]
```diff
@@ -104,14 +104,14 @@
...
 
-Here, we will generate data and conduct blind prediction challenges to expand the domain of applicability and robustness of these techniques. 
-The data we generate will span the range between systems which are tractable with current methods to those only slightly less complex than pharmaceutical drug targets covered by the NIH-funded D3R effort, which runs blind challenges on protein-ligand datasets obtained from the pharmaceutical industry.
-This systematic set of blind prediction challenges will expand the domain applicability, pushing free energy techniques into standard application in drug design.
+We propose to collected targeted experimental datasets, use them to conduct blind prediction challenges, and release curated benchmark sets in a manner designed to drive expansion of the domain of applicability and robustness of physical modeling techniques. 
+The data we generate will bridge the untamed chasm from systems tractable with current methodologies to the pharmaceutically-relevant drug targets featured in the NIH-funded D3R effort, which fields blind challenges using protein-ligand datasets from pharma.
```
Comment (davidlmobley) [inline latex/Specific Aims.tex:113]: I'll merge, though "untamed chasm" is perhaps a bit over the top; I'll come up with an alternative phrasing that backs off from this a bit. :)

File: latex/Specific Aims.tex [hunk 2]
```diff
@@ -104,14 +104,14 @@
 Computational screening techniques can do better than random selection of compounds, but they lack the accuracy to guide molecular design or optimization. 
 A new generation of physical techniques, alchemical free energy calculations, are poised to fill this void by providing a quantitative, predictive tool that can be used in multiple stages of the drug discovery pipeline, including lead optimization to improve affinity and selectivity or the retention of potency as other physical properties are optimized. 
 
-Recent success of alchemical methods sparked considerable enthusiasm, but the domain of applicability of these techniques is currently highly limited; for broad application and routine use, they need further evaluation, refinement, and development. 
+Recent success of alchemical methods in predicting accurate affinities has sparked considerable enthusiasm, but the domain of applicability of these techniques is currently highly limited; broad application and routine use will require further evaluation, refinement, and development.
```
Comment (davidlmobley) [inline latex/Specific Aims.tex:107]: I like the edit in the first part of the sentence; less a fan of the second half (moving from active voice to passive, though it DOES make it more concise). I'll try to come up with an alternate phrasing for the second part after merging.

File: latex/Specific Aims.tex [hunk 3]
```diff
@@ -104,14 +104,14 @@
...
-The data we generate will span the range between systems which are tractable with current methods to those only slightly less complex than pharmaceutical drug targets covered by the NIH-funded D3R effort, which runs blind challenges on protein-ligand datasets obtained from the pharmaceutical industry.
-This systematic set of blind prediction challenges will expand the domain applicability, pushing free energy techniques into standard application in drug design.
+We propose to collected targeted experimental datasets, use them to conduct blind prediction challenges, and release curated benchmark sets in a manner designed to drive expansion of the domain of applicability and robustness of physical modeling techniques. 
+The data we generate will bridge the untamed chasm from systems tractable with current methodologies to the pharmaceutically-relevant drug targets featured in the NIH-funded D3R effort, which fields blind challenges using protein-ligand datasets from pharma.
+Our systematic set of challenges aim to rapidly advance free energy techniques to the point of standard application in drug design.
```
Comment (davidlmobley) [inline latex/Specific Aims.tex:114]: "set ... aims". Will fix after merging.

File: latex/Specific Aims.tex [hunk 4]
```diff
@@ -131,7 +131,7 @@
 
 %{\bf \underline{Aim 2.} Measure binding of novel host-guest complexes for introductory ligand binding challenges.}\\
 {\bf \underline{Aim 2.} Measure affinities of drug-like compounds in supramolecular hosts to challenge quantitative models of binding in systems not plagued by major receptor sampling issues.}\\
-We will measure new host-guest binding free energies for cucurbiturils and deep-cavity cavitands, yielding further host-guest binding challenges which span between physical property prediction and protein-ligand binding. 
+We will measure new host-guest binding free energies (using cucurbiturils and deep-cavity cavitands as hosts) to field binding challenges which span in complexity between physical property prediction and protein-ligand binding.
```
Comment (davidlmobley) [inline latex/Specific Aims.tex:134]: Good, except not a fan of "span in complexity"; will rephrase slightly.

File: latex/Specific Aims.tex (hunk 2 above)
Comment (davidlmobley) [reply latex/Specific Aims.tex:107]: After thinking about it for a while, I guess I'll stick with your wording in the second clause. I still like mine marginally better, but I can't generate any alternatives to these two without making it longer, and yours is not bad.

File: latex/Specific Aims.tex (hunk 1 above)
Comment (davidlmobley) [reply latex/Specific Aims.tex:113]: (Also, I think we've already used the bridge metaphor too much in the prior paragraph, so I think we should avoid the word "bridge" here. Working on alternate wording.)

Comment (jchodera) [reply latex/Specific Aims.tex:113]: > I'll merge, though "untamed chasm" is perhaps a bit over the top; I'll come up with an alternative phrasing that backs off from this a bit. :)

Thanks! Couldn't think of anything else at the time.

**Comment by jchodera** [comment]
Thanks!

**Pull request merged**